packages/core/docs/react.stories.mdx

```diff
@@ -130,3 +119,20 @@ window.IntersectionObserver = jest.fn().mockReturnValue({
 <a href="https://github.com/vmware/clarity/tree/next/apps" target="_blank" rel="noopener">
   <cds-button>Example Apps</cds-button>
 </a>
+
+### Finding a Clarity element
+
+Because of the way web components are rendered, the synchronous
+`get` methods of RTL might not work, use the asynchronous `find` ones instead.
```

Comment:
👍

Reply:
I meant to update this and figured now was as good a time as any. I was going to mention puppeteer-testing-library or cypress-testing-library as alternative but don't know enough about using those with react to recommend it.